tools/input/section5.rs

```diff
@@ -104,6 +104,9 @@ unsafe fn parse_obu_header(
     buf_size -= 1;
     if extension_flag != 0 {
         buf = buf.offset(1);
```

Comment:
I think the check is supposed to go before the `buf++`.

Reply:
oops